mm2src/coins_activation/src/eth_with_token_activation.rs

```diff
@@ -228,6 +230,11 @@ impl PlatformWithTokensActivationOps for EthCoin {
         _storage: impl TxHistoryStorage + Send + 'static,
         _initial_balance: BigDecimal,
     ) -> AbortHandle {
-        todo!()
+        error!(
+            "tx_history is not allowed for {}. It works very slowly and requires a lot of disk space.",
+            self.ticker()
+        );
+        let (_, abort_handle) = abortable(async {});
+        abort_handle
```

Comment:
I think `start_history_background_fetching` should be modified to return a `Result<AbortHandle>` and return an error in this case for `ETH`. Right now if `tx_history` is set to `true` in `ETH` coin config this error will be logged which is not as good as returning an error on enabling the coin to tell the user explicitly that  `tx_history` can't be set for `ETH`

Reply:
I think this is better idea. What do you think? @artemii235 Please let me know if I should rollback commit 5975d6a

Reply:
`start_history_background_fetching` is called only when `tx_history` is true, which should be denied instead.

As an alternative, we can remove `tx_history` arg at all from `EthActivationV2Request` 🙂
Or return the error if it's set to `true` in some another place not touching `start_history_background_fetching`.

Reply:
I think removing `tx_history` is better idea since both cases `true` and `false` doesn't do anything at all. In that case, we don't need to notify the user that it's allowed or not and we don't need to log it. We can simply use `unimplemented!()` in `start_history_background_fetching` since it will not be executed because `tx_history()` will only return `false`. 

[a67c162](https://github.com/KomodoPlatform/atomicDEX-API/pull/1335/commits/a67c162660950b58e74588026ca09b592921a6de) should work the job.